issue/src/lib.rs

```diff
@@ -20,6 +20,7 @@ pub const HELP: Help = Help {
 Usage
 
     rad issue new [--title <title>] [--description <text>]
+    rad issue state <id> [--close | --open] [--solved]
```

Comment:
I would collapse it and make them states: `[--closed | --open | --solved]`, so there's no need to do `--close --solved`

Reply:
Doesn't look like this was solved :)

Reply:
😓 sorry yeah will add it asap

Reply:
Ok has been solved